docs/ingestion/native-batch-input-source.md

```diff
@@ -308,6 +321,7 @@ Sample specs:
 |uris|JSON array of URIs where the Azure objects to be ingested are located, in the form "azure://\<container>/\<path-to-file\>"|None|`uris` or `prefixes` or `objects` must be set|
 |prefixes|JSON array of URI prefixes for the locations of Azure objects to ingest, in the form "azure://\<container>/\<prefix\>". Empty objects starting with one of the given prefixes are skipped.|None|`uris` or `prefixes` or `objects` must be set|
 |objects|JSON array of Azure objects to ingest.|None|`uris` or `prefixes` or `objects` must be set|
+|filter|A wildcard filter for files.|None|no|
```

Comment:
IMHO this documentation can be improved. Do we have support for regex filtering? Wildcard seems just too limiting

Reply:
```suggestion
|filter|A wildcard filter for files. See [here](http://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/filefilter/WildcardFileFilter) for more information.|None|no|
```